### PR TITLE
Translate bertlogy.md into Chinese

### PR DESCRIPTION
**Pull request by wwwbai**
# What does this PR do?

I translated bertology.md into Chinese.This is my second attempt to contribute, and I hope to do better than last time.
## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

**Comment by Rocketknight1** [comment]
cc @stevhliu

**Review by stevhliu — COMMENTED** [review]
Minor fix to the toctree but otherwise LGTM! Can you ping or ask another contributor to review the translated content?

**Comment by wwwbai** [comment]
@statelesshz Considering the great value of your feedback on my previous translation work, would you be so kind as to review it？

**Comment by stevhliu** [comment]
Maybe @blueingman or @Isotr0py would also be interested in reviewing, given your ongoing translation efforts 🙂

File: docs/source/zh/bertology.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- BERT Rediscovers the Classical NLP Pipeline by Ian Tenney, Dipanjan Das, Ellie Pavlick:
+  https://arxiv.org/abs/1905.05950
+- Are Sixteen Heads Really Better than One? by Paul Michel, Omer Levy, Graham Neubig: https://arxiv.org/abs/1905.10650
+- What Does BERT Look At? An Analysis of BERT's Attention by Kevin Clark, Urvashi Khandelwal, Omer Levy, Christopher D.
+  Manning: https://arxiv.org/abs/1906.04341
+- CAT-probing: A Metric-based Approach to Interpret How Pre-trained Models for Programming Language Attend Code Structure: https://arxiv.org/abs/2210.04633
+
+
+为了助力这一新兴领域的发展，我们在BERT/GPT/GPT-2模型中增加了一些附加功能，以帮助人们访问内部表示，这些功能主要是基于Paul Michel的杰出工作进行调整的(https://arxiv.org/abs/1905.10650)：
```
Comment (blueingman) [inline docs/source/zh/bertology.md:26]:
```suggestion
为了助力这一新兴领域的发展，我们在BERT/GPT/GPT-2模型中增加了一些附加功能，方便人们访问其内部表示，这些功能主要借鉴了Paul Michel的杰出工作(https://arxiv.org/abs/1905.10650)：
```

File: docs/source/zh/bertology.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+除非适用法律要求或书面同意，否则按“按原样”分发的软件，无论是明示还是暗示的，都没有任何担保或条件。请参阅许可证以了解特定语言下的权限和限制。
+
+⚠️ 请注意，本文件虽然使用Markdown编写，但包含了特定的语法，适用于我们的doc-builder（类似于MDX），可能无法在您的Markdown查看器中正常渲染。
+
+-->
+
+# 基于BERT进行的相关研究（BERTology）
+
+有一个不断发展的研究领域，致力于探究像BERT这样的大规模transformer模型的内部工作机制（一些人称之为“BERTology”）。这个领域的一些典型案例包括：
```
Comment (blueingman) [inline docs/source/zh/bertology.md:15]:
```suggestion
当前，一个新兴的研究领域正致力于探索大规模 transformer 模型（如BERT）的内部工作机制，一些人称之为“BERTology”。以下是这个领域的一些典型示例：
```

**Review by blueingman — COMMENTED** [review]
I think this translation is quite good overall. My abilities are somewhat limited, so I’ve only made a few nitpicky suggestions.

File: docs/source/zh/bertology.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  Manning: https://arxiv.org/abs/1906.04341
+- CAT-probing: A Metric-based Approach to Interpret How Pre-trained Models for Programming Language Attend Code Structure: https://arxiv.org/abs/2210.04633
+
+
+为了助力这一新兴领域的发展，我们在BERT/GPT/GPT-2模型中增加了一些附加功能，以帮助人们访问内部表示，这些功能主要是基于Paul Michel的杰出工作进行调整的(https://arxiv.org/abs/1905.10650)：
+
+
+- 访问BERT/GPT/GPT-2的所有隐藏状态，
+- 访问BERT/GPT/GPT-2每个注意力头的所有注意力权重，
+- 检索注意力头的输出值和梯度，以便计算头重要性得分并进行剪枝，如前所述在https://arxiv.org/abs/1905.10650。
```
Comment (Isotr0py) [inline docs/source/zh/bertology.md:31]:
```suggestion
- 检索注意力头的输出值和梯度，以便计算头的重要性得分并对头进行剪枝，详情可见论文：https://arxiv.org/abs/1905.10650。
```

File: docs/source/zh/bertology.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+为了帮助您理解和使用这些功能，我们添加了一个具体的示例脚本：[bertology.py](https://github.com/huggingface/transformers/tree/main/examples/research_projects/bertology/run_bertology.py)，该脚本可以提取信息并对一个在GLUE上预训练的模型进行剪枝。
```
Comment (Isotr0py) [inline docs/source/zh/bertology.md:33]:
```suggestion
为了帮助您理解和使用这些功能，我们添加了一个具体的示例脚本：[bertology.py](https://github.com/huggingface/transformers/tree/main/examples/research_projects/bertology/run_bertology.py)，该脚本可以对一个在 GLUE 数据集上预训练的模型进行信息提取与剪枝。
```

**Review by Isotr0py — APPROVED** [review]
Overall looks good. Just some nits.

**Comment by wwwbai** [comment]
@Isotr0py @blueingman Thank you for your valuable suggestion!! I have added all the changes to my branch.

**Review by stevhliu — APPROVED** [review]
Looks good, thanks!

**Pull request merged**